plugins/phonenumber/lib_phonenumber.ecllib

```diff
@@ -0,0 +1,27 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2025 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+// Service definition
+EXPORT phonenumber := SERVICE : plugin('phonenumber'), library('phonenumber'), namespace('HPCCPhoneNumber'), CPP, PURE
```

Comment:
By convention it looks like we use `plugin('phonenumberplugin')` and `namespace('phonenumber')`